btree/btree.c

```diff
@@ -4275,6 +4276,9 @@ static void fkvv_dir_entry_delete(const struct nd *node, int idx)
 	total_size = sizeof(*dir) * (h->fkvv_dir_entries - idx - 1);
 
 	m0_memmove(&dir[idx], &dir[idx + 1], total_size);
+	h->fkvv_dir_entries--;
+	dir[h->fkvv_dir_entries - 1].val_offset     -= sizeof(*dir);
+	dir[h->fkvv_dir_entries - 1].alloc_val_size += sizeof(*dir);
```

Comment:
Are you sure this code is correct ? Please review again.

Reply:
I am maintaining the last free fragment at index  h->fkvv_dir_entries - 1. The last free fragment is the free region between last valid key and start of the directory in the case of key.  In case of value, the last free region will be between end of directory and last valid value.
![image](https://user-images.githubusercontent.com/68048778/167829619-7a945c75-a65a-4f2f-8c90-7f515041eeff.png)

So, if we delete one directory entry, the size of last free fragment will increase.